lbz/authz.py

```diff
@@ -138,13 +141,17 @@ def decorator(func: Callable):
         @wraps(func)
         def wrapped(self, *args, **kwargs):
             authorization_header = self.request.headers.get("Authorization")
-            if not authorization_header:
+            authorization_scope = None
+            if hasattr(self, "get_guest_authorization"):
```

Comment:
So if the resource has a `get_guest_authorization` method it will never use the policy from the auth header? I don't think that's right?

Also I think we should add a test case to the `test_authz_decorator` file